.kokoro/docs.sh

```diff
@@ -2,17 +2,9 @@
 
 # Copyright 2018 Google LLC
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Oh my - so all of the files in `.kokoro`, `.prettierignore`, `.mocharc.js`, `.jsdoc.js` - those will continue to have an Apache license.  They are brought in as part of a set of templates that are consistent across all of our nodejs libraries.  None of these files are shipped with the product code in the `npm` package, so I don't think this is going to cause us any trouble.  We should revert these.

Reply:
Ok no prob

Reply:
Just calling out that we'll continue to have the discrepancies mentioned in #399 ...